libkbfs/interfaces.go

```diff
@@ -1205,20 +1218,23 @@ type KeyOps interface {
 type Prefetcher interface {
 	// PrefetchBlock directs the prefetcher to prefetch a block.
 	PrefetchBlock(block Block, blockPtr BlockPointer, kmd KeyMetadata,
-		priority int) error
+		priority int) (doneCh, errCh <-chan struct{}, err error)
 	// PrefetchAfterBlockRetrieved allows the prefetcher to trigger prefetches
 	// after a block has been retrieved. Whichever component is responsible for
 	// retrieving blocks will call this method once it's done retrieving a
 	// block. Returns a channel that is closed once the all the underlying
 	// prefetches are complete.
 	PrefetchAfterBlockRetrieved(b Block, blockPtr BlockPointer,
-		kmd KeyMetadata) <-chan struct{}
+		kmd KeyMetadata) (doneCh, errCh <-chan struct{}, numBlocks int)
```

Comment:
The comment here and for `PrefetchBlock` should be updated to describe the new return channels.